File: Python/11-20/e_16.py
# Escribe las instrucciones en python que permitan ordenar una cadena de caracteres en orden alfabético ascendente. 
# Para probar, vamos a tomar la cadena c = "finlandia".

# Cadena de caracteres
c = "finlandia"

# Ordenar la cadena en orden alfabético ascendente
c_ordenada = ''.join(sorted(c))

# Mostrar la cadena ordenada
print("Cadena ordenada:", c_ordenada)


# En este código, utilizamos la función sorted() para obtener una lista de caracteres ordenados,
# luego usamos join() para convertir esa lista de caracteres nuevamente en una cadena. La salida esperada sería "aadfiilnn".

File: JS/11-20/e_16.js
// Escribe las instrucciones en JS que permitan ordenar una cadena de caracteres en orden alfabético ascendente. 
// Para probar, vamos a tomar la cadena c = "finlandia".


// Cadena de caracteres
var c = "finlandia";

// Convertir la cadena en un array de caracteres, ordenar y luego unir de nuevo en una cadena
var c_ordenada = c.split('').sort().join('');

// Mostrar la cadena ordenada
console.log("Cadena ordenada:", c_ordenada);

// En este código, split('') convierte la cadena en un array de caracteres, sort() ordena los caracteres,
// finalmente, join('') une los caracteres ordenados en una cadena nuevamente. La salida esperada sería "aadfiilnn".